sale_order_type/README.rst

```diff
@@ -16,7 +20,7 @@ Usage
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
And in Usage section you can add something like:

#. Go to *Sales > Sales Orders* and create a new one.
#. Select the new type you are created before and all settings will be 
   propagated.
#. Or you can define a type for a particular partner if you go to *Sales &
   Purchases* and set a sale order type.